### PR TITLE
Add support for influx escaping rules

### PR DESCRIPTION
**Pull request by davecromberge**
Escaping rules for line format are here:https://docs.influxdata.com/influxdb/v0.13/write_protocols/write_syntax/

### CODE REVIEW

File: src/dp_influx.erl [hunk 1]
```diff
@@ -52,10 +51,19 @@ parse_tag(<<"=", V/binary>>, K) ->
 parse_tag(<<C, R/binary>>, K) ->
     parse_tag(R, <<K/binary, C>>).
 
+%% Time format values such as \" 0:00 \" are ignored
+parse_metrics(<<"\" ", _F:4/binary, "\"", R/binary>>, _Metric, Metrics, M) ->
+    parse_metrics(R, <<>>, Metrics, M);
```
Comment (Licenser) [inline src/dp_influx.erl]: that doesn't seem like a good approach, if influx sends strings they might be more then a space and 4 characters, or even be escaped. I fear the real approach here is to pars strings and ignore them all togehter

**Comment by davecromberge** [comment]
Thanks for the review, I agree with you, and have changed the approach.  Let me know if this is what you had in mind.

**Comment by Licenser** [comment]
I think it's a bit more complicated then that my gut feeling tells me that if influx allow strings they also allow escaping in strings, that needs to be considered (probably not so bad). And I don't think a leading space is required in a string it could just be `"abc"` not nessessarily `" abc"`

**Comment by davecromberge** [comment]
Yes I see there are other escaping rules in the measurement names that we need to adhere to as well, will spend some time adding tests similar to their [escaping tests](https://github.com/influxdata/influxdb/blob/master/models/points_test.go#L637).

**Comment by Licenser** [comment]
that looks awesome!

**Pull request merged**